core/Cargo.toml

```diff
@@ -217,6 +217,9 @@ services-webdav = []
 services-webhdfs = []
 services-yandex-disk = []
 
+# If enabled, will use undocumented workflow with azdls API to support list with start after
+azdls-list-with-start-after = []
```

Comment:
Using a compile-time feature to control this doesn’t seem ideal to me. Instead, I’m considering providing a configuration option, such as `AzdlsConfig::enable_list_start_after()`. We can clearly indicate that this is an undocumented API and offer additional context as needed.